src/tests/Interop/Swift/ErrorHandling.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.Swift;
+using System.Text;
+using Xunit;
+
+public class ErrorHandlingTests
+{
+    private const string SwiftLib = "libErrorHandling.dylib";
+
+    [DllImport(SwiftLib, EntryPoint = "$s13ErrorHandling05setMyA7Message5bytes6lengthySPys5UInt8VG_SitF")]
+    public static extern void SetErrorMessage(byte[] strBytes, int length);
+
+    [UnmanagedCallConv(CallConvs = new Type[] { typeof(CallConvSwift) })]
+    [DllImport(SwiftLib, EntryPoint = "$s13ErrorHandling018conditionallyThrowA004willD0SiSb_tKF")]
+    public unsafe static extern nint conditionallyThrowError(bool willThrow, SwiftError* error);
+
+    [DllImport(SwiftLib, EntryPoint = "$s13ErrorHandling05getMyA7Message4fromSPys5UInt8VGSgSPyAA0dA0OG_tF")]
+    public static extern IntPtr GetErrorMessage(IntPtr handle);
+
+    [Fact]
+    public unsafe static int TestEntryPoint()
```

Comment:
This is an unhelpful test name. What "entry point"? Based on the test below, it seems this is for exceptions, but there are multiple cases being tested here.

Reply:
You are right. I've added different test cases.